python/Motion Platform Inverse Kinetics.ipynb

```diff
@@ -95,6 +95,15 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "Set up as matrix equation, solve and celebrate!!"
```

Comment:
This is the actual line I changed in my python notebook for the demo pull request.